Formula/n/nubesgen.rb

```diff
@@ -0,0 +1,28 @@
+class Nubesgen < Formula
+  desc "Kickstart your project on Azure in minutes!"
+  homepage "https://nubesgen.com/"
+  url "https://github.com/microsoft/NubesGen/releases/download/v0.17.0/nubesgen-cli-0.3.0-SNAPSHOT.jar"
+  version "0.17.0"
+  sha256 "6e8ee4fc4bc887f1741bcf758475f50f7cf0f706e91c74a80c541e7f9da2f06c"
+  license "MIT"
+
+  depends_on "azure-cli"
+  depends_on "gh"
+  depends_on "openjdk"
+
+  def install
+    libexec.install "nubesgen-cli-0.3.0-SNAPSHOT.jar" => "nubesgen-#{version}.jar"
+
+    # Force using the openjdk dependency
+    (bin/"nubesgen").write <<~EOS
+      #!/bin/bash
+      export JAVA_HOME="#{Language::Java.java_home_env[:JAVA_HOME]}"
+      exec "${JAVA_HOME}/bin/java" -jar "#{libexec}/nubesgen-#{version}.jar" "$@"
+    EOS
```

Comment:
```suggestion
    # Force using the openjdk dependency
    bin.write_jar_script libexec/"nubesgen-#{version}.jar", "nubesgen"
```

Reply:
Hello Sean, thank you for your review. I think I tried what you’re suggesting but if I recall correctly the `write_jar_script` command generates a `JAVA_HOME` which can be overridden by the user’s `JAVA_HOME`. Currently the Nubesgen jar requires Java 11. It will break if the user’s Java version is older than 11. that’s why I’m forcing the formula to run with the openjdk version by making the `JAVA_HOME` variable not overridable so that nubesgen always runs with the dependent openjdk version.

Is there a parameter for the `write_jar_script` which generates a non overridable `JAVA_ENV`?

Reply:
Yes, I just tested and your suggestion generates the following shell script:

```
#!/bin/bash
export JAVA_HOME="${JAVA_HOME:-/opt/homebrew/opt/openjdk/libexec/openjdk.jdk/Contents/Home}"
exec "${JAVA_HOME}/bin/java"  -jar "/opt/homebrew/Cellar/nubesgen/0.17.0/libexec/nubesgen-0.17.0.jar" "$@"
```

Which doesn't force using the openjdk dependency.

What I want is to generate the following script:

```
#!/bin/bash
export JAVA_HOME="/opt/homebrew/opt/openjdk/libexec/openjdk.jdk/Contents/Home"
exec "${JAVA_HOME}/bin/java" -jar "/opt/homebrew/Cellar/nubesgen/0.17.0/libexec/nubesgen-0.17.0.jar" "$@"
```

Do you mind keeping the code I initially suggested unless there is a shorter way of achieving the same result?

Reply:
Gentle ping on this PR :)